contents/handbook/people/compensation.mdx

```diff
@@ -83,18 +83,30 @@ If you are nomading, we will set your location factor for the place that you are
 ​
 It’s important to us that all PostHog employees can feel invested in the company’s success. Every one of us plays a critical role in the business and deserves a share in the companies success as we grow. When employees perform well, they contribute to the business doing well, and therefore should share a part of the increased financial value of the business.
 
-As part of your compensation, you will receive share options in the company with a standard 1-year cliff, 4-year vest. Broadly, the amount of options will depend on the Level as per the Experience Factor. We may change this policy from time to time depending on our rate of hiring - e.g. if we had a gap in hiring for an extended period, we would adjust this.
+As part of your compensation, you will receive share options in the company. Broadly, the amount of options will depend on the Level as per the Experience Factor. We may change this policy from time to time depending on our rate of hiring - e.g. if we had a gap in hiring for an extended period, we would adjust this.
 
-Whilst the terms of options for _any company_ could vary if we were ever acquired, we have set them up with the following key terms:
+Whilst the terms of options for _any company_ could vary if we were ever acquired, we have set them up with the following key terms which we believe are industry-leading in their friendliness to employees:
 
+-   Standard 4-year vesting with a 1-year cliff
 -   10 years to exercise your options in the event that you leave PostHog
 -   Double trigger acceleration, which means if you are let go or forced to leave due to the company being acquired, you receive all of your options at that time
 -   Vesting starts from your start date (not after a "probation period" or similar)
 -   For UK-based team members, our options are part of the [EMI share options scheme](https://www.gov.uk/tax-employee-share-schemes/enterprise-management-incentives-emis), which is tax-advantaged
 
-It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog. While we can commit to a particular percentage, we cannot commit to a particular strike price when offering share options, as the valuations are done by a third party and can vary depending on where we are in our funding cycle. 
+It can take time to approve options, as it requires a board meeting and company valuation. We can clarify the likely time frame at the time we're hiring you. In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog. While we can commit to a particular percentage, we cannot commit to a particular strike price when offering share options, as the valuations are done by a third party and can vary depending on where we are in our funding cycle.
```

Comment:
> In any case, you will not be disadvantaged as vesting will always start from when you joined PostHog

This isn't strictly true as if the strike price changes between joining and issuing, you lose out. I know you allude to this in the next bit, but I think I wouldn't make this claim.